CHANGES.txt

```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+Version 0.2.4, 11-Mar-2015.
+
+* Updating Mixed Layer Depth functions
+* Examples included
```

Comment:
No need for this line, in theory examples a given and not a feature that needs announcing :grimacing: